src/pages/tasks/TaskDescriptionPage.js

```diff
@@ -39,9 +39,13 @@ function TaskDescriptionPage(props) {
 
     const submit = useCallback(
         (values) => {
-            // Set the description of the report in the store and then call Task.editTaskReport
-            // to update the description of the report on the server
-            Task.editTaskAndNavigate(props.report, {description: values.description});
+            if (values.description !== props.report.description) {
```

Comment:
@hoangzinh If we add a new line, though there is no change in description, it is marked as unread.

Reply:
@hoangzinh With the latest commits, If I save description without any changes, it is marked as unread. Can you please test it in your end?

Reply:
@hoangzinh Could you please try with input with multiple lines(I tested with input with two lines)?